src/current/_data/versions.csv

```diff
@@ -15,4 +15,4 @@ v23.1,2023-05-15,2024-05-15,2024-11-15,23.1.11,23.1.12,2023-11-13,2024-11-13,202
 v23.2,2024-02-05,2025-02-05,2025-08-05,23.2.6,23.2.7,2024-07-08,2025-07-08,2026-07-08,v23.1,release-23.2
 v24.1,2024-05-20,2025-05-20,2025-11-20,N/A,N/A,N/A,N/A,N/A,v23.2,release-24.1
 v24.2,2024-08-12,2025-02-12,N/A,N/A,N/A,N/A,N/A,N/A,v24.1,release-24.2
-v24.3,N/A,N/A,N/A,N/A,N/A,N/A,N/A,N/A,v24.2,master
+v24.3,N/A,N/A,N/A,N/A,N/A,N/A,N/A,N/A,v24.2,release-24.3

```

Comment:
This may need to be reverted.

Reply:
I've tried both `release-24.3` and `master` locally and I can't get this to build.